readme.md

```diff
@@ -40,50 +41,56 @@ Hello, Node!
 
 * **Your Answer:**
 
----
+## "cat classmates.txt" for reading the content of the file* ##*"echo classmates.txt" for updating and in this case for appending new content to the file.\_ \_"rm classmates.txt" for deleting a file.
 
 - [ ] Consider the above and then look back through the [fs module](https://nodejs.org/api/fs.html) documentation.
 
 * **Question:** What methods represent each CRUD action?
 
 * **Your Answer:**
 
----
+## C: fs.writeFile(),fs.appendFile(),fs.open()
+
+## R: fs.readFile()
+
+## U: fs.appendFile()
+
+## D: fs.unlink()
 
 - [ ] Take a look at the following two methods: [fs.readFile()](https://nodejs.org/api/fs.html#fs_fs_readfile_path_options_callback) and [fs.readFileSync()](https://nodejs.org/api/fs.html#fs_fs_readfilesync_path_options).
 
 * **Question:** What is the difference between these two methods?
 
-* **Your Answer:**
-
----
+* **Your Answer:** ##_fs.readFile() is asynchron and does have a callback method but fs.readFileSync doesn't have a callback methode._
 
 - [ ] Take a look at [fs.writeFile()](https://nodejs.org/api/fs.html#fs_fs_writefile_file_data_options_callback) and [fs.appendFile()](https://nodejs.org/api/fs.html#fs_fs_appendfile_path_data_options_callback).
 
 * **Question:** Describe the difference between these two methods.
 
 * **Your Answer:**
 
----
+## fs.writeFile() re-writes whole file but fs.appendFile() adds data to the existing file instead of overwriting it.
 
 - [ ] Imagine you want to edit the middle of a file. You can use the [fs module](https://nodejs.org/api/fs.html) and the JavaScript language.
 
 * **Question:** How would you do so?
 
 * **Your Answer:**
 
----
+## We should read the pets.json file and parse it as a JSON then push our changes as a json object turn ut back to string and then write the file
 
 - [ ] In Node, you'll have access to a global variable called `__dirname`. Add the following to your `index.js` file.
   ```js
-  const path = require('path')
-  const petsFile = path.join(__dirname, 'data', 'pets.json')
+  const path = require("path");
+  const petsFile = path.join(__dirname, "data", "pets.json");
   ```
 
 * **Question:** Describe what is happening in the above code.
 
 * **Your Answer:**
 
+## By using path and \_\_dirname and at the end logging petsFile we can see the pets.json file's path in the console.
```

Comment:
That's true. It's also constructing a dynamic path that will work independent of file systems.